### PR TITLE
Added code for pull server removal

### PR DESCRIPTION
**Pull request by hahndorf**
This update fixes the following issues: #23, #26, #27, #28 and #29 
I added an integration test with a few basic tests.
Because this update adds new logic and removes existing code (#29) someone should have a good look.

### CODE REVIEW

**Comment by hahndorf** [comment]
I spent hour trying to get my tests working on AppVeyor, in the end I gave up. DSC can't find my resources whatever I do.

File: DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1 [hunk 1]
```diff
@@ -96,8 +96,8 @@ function Set-TargetResource
         # Port number of the DSC Pull Server IIS Endpoint
         [Uint32]$Port = $( if ($IsComplianceServer) { 7070 } else { 8080 } ),
 
-        # Physical path for the IIS Endpoint on the machine (usually under inetpub/wwwroot)                            
-        [string]$PhysicalPath = "$env:SystemDrive\inetpub\wwwroot\$EndpointName",
```
Comment (rayhayes) [inline DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1]: I see that the installation does go to C:\inetpub\wwwroot\psdscpullserver currently. Why is it being changed?

Comment (hahndorf) [inline DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1]: See Issue #26 - C:\inetpub\wwwroot is not the location to place files for a new web site, it is for files of the 'Default Web Site'. The old location mixes files of two totally separate application and may make the pull server files available to users who should never see it. We can discuss this in issue #26

**Comment by hahndorf** [comment]
Do you still need any input from me? I'm note sure why there is a `Waiting for the author` tag

File: DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1 (hunk 1 above)
Comment (rayhayes) [inline DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1]: OK. I'm fine with all the changes.

**Pull request merged**